Donation-platform/src/main/webapp/browse.js

```diff
@@ -165,4 +96,16 @@ function updatePageWithCharities(tagName, charities)
     const numberPerPage = 9;
     let pagination = new Pagination(charities, tagName, pageList, currentPage, numberPerPage);
     pagination.load();
+}
+/**
+ * Sorts objects by name.
+ */
+function compare( a, b ) {
+  if ( a.name < b.name ){
+    return -1;
+  }
+  if ( a.name > b.name ){
+    return 1;
+  }
+  return 0;
```

Comment:
you can write this as 

function compare(a, b) {
 return (a.name < b.name)
}

Reply:
resolved.